app/components/page-components/Chatbot/ChatbotDefaultCommands.vue.ts

```diff
@@ -3,19 +3,21 @@ import { Component } from 'vue-property-decorator';
 import { IDefaultCommand } from 'services/chatbot';
 import { $t } from 'services/i18n';
 import CollapsibleSection from 'components/shared/CollapsibleSection.vue';
+import { mapValues, pickBy, keys } from 'lodash';
```

Comment:
can you import these separately so we can properly tree-shake the lodash library? also `keys` is available to us as simply `Object.keys`, no need for a helper here

Reply:
import separately as in import {mapvalues} from 'lodash' and {pickBy} from 'lodash' both on separate liens?